content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -205,6 +205,19 @@ field in the quota spec.
 
 A quota is matched and consumed only if `scopeSelector` in the quota spec selects the pod.
 
+Quota for the following resources can be scoped to Priority class, with the use of `scopeSelector`:
+
+* `limits.cpu`
+* `limits.memory`
+* `limits.ephemeral-storage`
+* `count/pods`
+* `requests.cpu`
+* `requests.memory`
+* `requests.ephemeral-storage`
```

Comment:
just curious, how did you find this list?

Reply:
From

https://github.com/kubernetes/kubernetes/blob/98814409bab818d942e3c341d76eb594f12675a5/pkg/apis/core/validation/validation.go#L5424

The resource name should satisfy:

- NOT a standard resource, or
- In the list of `pods`, `cpu`, `memory`, `limits.cpu`, `limits.memory`, `requests.cpu`, `requests.memory`.

So in other words, the legit resource name should be: `pods`, `cpu`, `memory`, `limits.cpu`, `limits.memory`, `requests.cpu`, `requests.memory`, plus extended resource (e.g., `nvidia.com/gpu`).

@m9a, where did you find that `[requests|limits].ephemeral-storage` is allowed?

Reply:
Does the LocalStorageCapacityIsolation feature gate change that behavior maybe? (I haven't checked)

Reply:
`LocalStorageCapacityIsolation` is beta since 1.10, so it's enabled by default.